nautobot_data_validation_engine/api/nested_serializers.py

```diff
@@ -0,0 +1,89 @@
+"""
+API serializers
+"""
+from rest_framework import serializers
+
+from nautobot.core.api import WritableNestedSerializer
+
+from nautobot_data_validation_engine.models import (
+    MinMaxValidationRule,
+    RegularExpressionValidationRule,
+    RequiredValidationRule,
+    UniqueValidationRule,
+)
+
+
+__all__ = [
+    "NestedRegularExpressionValidationRuleSerializer",
+    "NestedMinMaxValidationRuleSerializer",
+    "NestedRequiredValidationRuleSerializer",
+    "NestedUniqueValidationRuleSerializer",
+]
+
+
+class NestedRegularExpressionValidationRuleSerializer(WritableNestedSerializer):
+    """Serializer for `RegularExpressionValidationRule` objects."""
+
+    url = serializers.HyperlinkedIdentityField(
+        view_name="plugins-api:nautobot_data_validation_engine-api:regularexpressionvalidationrule-detail"
```

Comment:
Nit: should we change the API URL pattern naming convention so that it uses `plugins-api:nautobot-data-validation-engine-api` instead (consistent use of `-` instead of mixing `_` and `-`?)

Reply:
Seems like it's not possible without changing Nautobot code:
https://github.com/nautobot/nautobot/blob/develop/nautobot/utilities/utils.py#L117

Reply:
Ah right, I forgot this was based on app_label. Thanks for checking!